frontend/customization/config/global.json

```diff
@@ -10,10 +10,8 @@
   "apiUrl": "https://geotrekdemo.ecrins-parcnational.fr/api/v2",
   "reservationPartner": "parcdesecrins",
   "reservationProject": "parcdesecrins",
```

Comment:
```suggestion
```
I think we should remove these two lines

Reply:
Yes indeed.